enrolments/serializers.py

```diff
@@ -31,3 +32,28 @@ class ClassListSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Class
         fields = ["id", "name", "facilitator_id"]
+
+
+class ClassDetailSerializer(serializers.HyperlinkedModelSerializer):
+    families = serializers.SerializerMethodField()  # defaults to get_families
+
+    class Meta:
+        model = Class
+        fields = [
+            "id",
+            "name",
+            "session_id",
+            "facilitator_id",
+            "attendance",
+            "families",
+        ]
+
+    def get_families(self, obj):
+        e_set = Enrolment.objects.filter(enrolled_class=obj)
+        return [
+            FamilyAttendanceSerializer(
+                enrolment.family, read_only=True, context={"request": None}
+            ).data
+            for enrolment in e_set
+            if enrolment.active == True
```

Comment:
Added logic to only get families who have an active enrolment here. The logic for this should be in the ClassDetail serializer right?